Makefile

```diff
@@ -208,7 +207,10 @@ endif
 			ln -srf $(DESTDIR)$(pkglibdir)/modules.d/98dracut-systemd/$$i $(DESTDIR)$(systemdsystemunitdir); \
 			ln -sf ../$$i \
 			$(DESTDIR)$(systemdsystemunitdir)/initrd.target.wants/$$i; \
-		done \
+		done; \
+		ln -srf $(DESTDIR)$(pkglibdir)/modules.d/99initqueue/dracut-initqueue.service $(DESTDIR)$(systemdsystemunitdir); \
+		ln -sf ../dracut-initqueue.service \
```

Comment:
Hello, can you please elaborate on this symlink?

Reply:
This creates symlinks that were previously done by the `for` loop above.